src/generators/elmFunctionDeclarationInformation.ts

```diff
@@ -0,0 +1,12 @@
+import * as ESTree from "estree";
+
+export function* elmFunctionDeclarationInformation(
+  statement: ESTree.Statement
+) {
+  if (statement.type === "FunctionDeclaration") {
+    yield {
+      name: statement.id?.name ?? "anonymous",
```

Comment:
The `id` field may be `null` if no name is given to the function (anonymous function) according to TS and so I supply the `"anonymous"` fallback name in such a case.